packages/muelu/test/scaling/CMakeLists.txt

```diff
@@ -415,22 +415,17 @@ IF (${PACKAGE_NAME}_ENABLE_Experimental AND ${PACKAGE_NAME}_ENABLE_Tpetra)
       COMM serial mpi
       )
 
-    TRIBITS_ADD_EXECUTABLE(
-      MatvecKernelDriver
-      SOURCES MatvecKernelDriver.cpp
-      COMM serial mpi
-      )
-  
-
-    TRIBITS_ADD_TEST(
-      MatvecKernelDriver
-      NAME "MatvecKernelDriver"
-      ARGS "--linAlgebra=Tpetra"
-      NUM_MPI_PROCS 1
-      COMM serial mpi
-      )
   ENDIF()
 ENDIF()
 
 
+# remove this from Experimental it requires Galeri + Tpetra
+IF (${PACKAGE_NAME}_ENABLE_Tpetra)
```

Comment:
Galeri is optional (surprisingly! see `Dependencies.cmake` above), so you'll also need to protect this with `${PACKAGE_NAME}_ENABLE_Galeri`.

Reply:
Unless I screwed up, this is inside the larger Galeri statement.